cpp/modmesh/view/RApplication.cpp

```diff
@@ -62,133 +62,6 @@ RApplication * RApplication::instance()
 
 RApplication & RApplication::setup()
 {
-    /* TODO: parse arguments */
```

Comment:
Refactor the main window setup code to make it more maintainable.

Reply:
I like the new one